### PR TITLE
Sdk v2

### PR DESCRIPTION
**Pull request by wrighbr**

### CODE REVIEW

File: models/user.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ var PathUsers = "/users"
 
 //
 type UserBody struct {
-	Localation      string `json:"localation,omitempty"`
+	Location      string `json:"location,omitempty"`
```
Comment (Gaardsholt) [inline models/user.go:7]: Just do be annoying, can we fix the indents :)
You should actually be able to just run `go fmt ./...`

**Review by Gaardsholt — APPROVED** [review]
LGTM - but I didn't look through the upgrade guide to see if we are using deprecated or removed stuff
https://www.terraform.io/docs/extend/guides/v2-upgrade-guide.html

**Review by brondum — APPROVED** [review]
LGTM.
Fail fast right :D  (if it dont work, that is)

File: models/user.go (hunk 1 above)
Comment (brondum) [reply models/user.go:7]: @wrighbr I think your VSCode linter should do this on every save, maybe run the setup script again ?

**Comment by wrighbr** [comment]
thanks god for automated acceptance testing

File: models/user.go (hunk 1 above)
Comment (wrighbr) [reply models/user.go:7]: all good I updated it using codespell to fix all my great spelling mistakes

**Pull request merged**